src/webview/wkwebview/mod.rs

```diff
@@ -650,6 +654,8 @@ impl Drop for InnerWebView {
       // So we need to override the process by navigating to `about:blank`.
       self.navigate("about:blank");
 
+      let _: () = msg_send![self.manager, removeAllScriptMessageHandlers];
```

Comment:
This is pretty similar to what i tried, but it has the same problem. If you try to close a window the app straight up panics
> unrecognized selector sent to instance

Idk from my testing it seemed like doing it in `drop` was too late :thinking:

Reply:
hmm, this is not occurred in my environment...
Could you show me to reproduce code?

Reply:
You can use this exact code (a fresh clone of your fork panics on my end too).
The easiest to use is wry's multi_window example and spawning and closing a window immediately causes a panic with a reference to this line of code.

Reply:
Aaahh i see why. This api was added in macos 11 and i primarily tested in 10.15

Reply:
Soo changing it back to `removeScriptMessageHandlerForName` doesn't panic anymore (it did in my first tests, guess i was doing something wrong.). But neither your PR nor changing it to remove `"ipc"` has an effect.

Reply:
Oh I see.
Thank you for your description :)
I will fix it.